draft-packages/table/KaizenDraft/Table/styles.scss

```diff
@@ -77,6 +77,8 @@ $row-height: 60px;
 
 .headerRowCellButton {
   @include button-reset;
+  text-decoration: none;
+  color: $kz-color-wisteria-800;
```

Comment:
How about using `.anchorReset` from this same file?

Reply:
Ah thanks! I didn't realise we had that. Updated now